<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsservStreamControlPanel</class>
 <widget class="QMainWindow" name="AsservStreamControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AsservStreamControlPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>551</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>RÃ©glages asserv</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>511</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_vitesse_gauche">
      <attribute name="title">
       <string>vitesse roue gauche</string>
      </attribute>
      <widget class="QPushButton" name="vitesse_gauche_update_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vitesse Kp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vitesse Ki</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="vitesse_gauche_Kp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="vitesse_gauche_Ki">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Range</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="vitesse_gauche_range">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_vitesse_droite">
      <attribute name="title">
       <string>vitesse roue droite</string>
      </attribute>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Range</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vitesse Ki</string>
       </property>
      </widget>
      <widget class="QPushButton" name="vitesse_droite_update_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="vitesse_droite_Ki">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="vitesse_droite_Kp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vitesse Kp</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="vitesse_droite_range">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_distance">
      <attribute name="title">
       <string>Distance</string>
      </attribute>
      <widget class="QDoubleSpinBox" name="distance_Kp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Distance Kp</string>
       </property>
      </widget>
      <widget class="QPushButton" name="distance_update_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_angle">
      <attribute name="title">
       <string>Angle</string>
      </attribute>
      <widget class="QDoubleSpinBox" name="angle_Kp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Angle Kp</string>
       </property>
      </widget>
      <widget class="QPushButton" name="angle_update_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>70</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_motor_control">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Motor control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="motor_enable_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QPushButton" name="motor_disable_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>721</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Commandes </string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>260</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne position:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pos_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send position command</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>701</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne vitesse avancement robot:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="robot_speed_lin_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>701</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_speed_lin_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send robot speed command</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>200</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>mm/sec</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>701</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="angle_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>320</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne angle:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="angle_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>320</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send angle command</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>140</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad/sec</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne vitesse angulaire robot:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="robot_speed_ang_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_speed_ang_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>140</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send robot speed command</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>701</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="left_whell_speed_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send wheel speed command</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="left_wheel_speed_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne vitesse roue gauche:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad/sec</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>701</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="right_wheel_speed_cmd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consigne vitesse roue droite:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right_whell_speed_cmd_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send wheel speed command</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>rad/sec</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dist_angle_controlmotor_control_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>distance / angle control</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="distAngle_enable_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QPushButton" name="distAngle_disable_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
